src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
 
-export default function VeteranIDCard() {
-  return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
-  );
+const FONT_FAMILY = 'Arial, Helvetica, sans-serif';
+// sets the aspect ratio of svg
+const VIEW_BOX = '0 0 2048 1284';
+// height of font top to bottom bounding box in the above viewbox
+const LARGE_FONT_SIZE = 90;
+const SMALL_FONT_SIZE = 60;
+
+const imagePaths = {
+  'Coast Guard': '/img/vic-cg-emblem.png',
+  'Air Force': '/img/vic-air-force-coat-of-arms.png',
+  Army: '/img/vic-army-symbol.png',
+  Navy: '/img/vic-navy-emblem.png',
+  'Marine Corps': '/img/vic-usmc-emblem.png',
+  previewBackground: '/img/vic-preview-background.svg',
+  VASeal: '/img/vic-va-seal.png'
+};
+
+function getIsLongName(name) {
+  try {
+    return document
+      .createElement('canvas')
+      .getContext('2d')
+      .measureText(name)
+    // manually determined max width
+      .width > 120;
+  } catch (e) {
+    return true;
+  }
 }
+
+const VeteranIDCard = ({
+  veteranFirstName,
+  veteranLastName,
+  veteranBranch,
+  veteranID
+}) => (
+  // svg preserves aspect ratio
+  <svg
+    fill={'white'}
+    fontFamily={FONT_FAMILY}
+    preserveAspectRatio={'xMidYMid meet'}
+    stroke={'none'}
+    viewBox={VIEW_BOX}>
+
+    <image href={imagePaths.previewBackground} width={'100%'}/>
+
+    <image height={'28.5%'} href={imagePaths.VASeal} x={'6.5%'} y={'7.3%'}/>
+    <text fontSize={LARGE_FONT_SIZE} x={'26.3%'} y={'24.3%'}>
+      Veteran Identification Card
+    </text>
+
+    {
+      getIsLongName(`${veteranFirstName} ${veteranLastName}`) ?
+        [
+          (
+            <text fontSize={LARGE_FONT_SIZE} key={'firstName'} x={'6.5%'} y={'50.6%'}>
+              {veteranFirstName}
+            </text>
+          ),
+          (
+            <text fontSize={LARGE_FONT_SIZE} key={'lastName'} x={'6.5%'} y={'58.8%'}>
+              {veteranLastName}
+            </text>
+          )
+        ] : (
+          <text fontSize={LARGE_FONT_SIZE} x={'6.5%'} y={'55.2%'}>
+            {`${veteranFirstName} ${veteranLastName}`}
+          </text>
+        )
+    }
```

Comment:
This spacing is kind of gross but wanted to make sure it was clear the first condition was returning an array of elements.